### PR TITLE
Specify python-version using str in workflow files

### PR DESCRIPTION
**Pull request by himkt**
To prepare Python 3.10, I updated `python-version` to use quotation marks.

🔗 https://github.com/optuna/optuna/pull/3353/commits/2e4602b8e3a3a7c4acb6251be221f3574aa99841

### CODE REVIEW

**Comment by himkt** [comment]
The failure of fastai will be fixed in https://github.com/optuna/optuna-examples/pull/93.

**Comment by hvy** [comment]
Thanks for the PR!
#93 has been merged and is causing a conflict. Could you resolve it?

File: .github/workflows/wandb.yml [hunk 1]
```diff
@@ -35,4 +35,4 @@ jobs:
         wandb offline
         python wandb/wandb_simple.py
       env:
-        OMP_NUM_THREADS: 1
```
Comment (hvy) [inline .github/workflows/wandb.yml:38]: 👍

**Comment by hvy** [comment]
Rerunning failed `allennlp` since https://github.com/optuna/optuna/pull/3413 was just merged.

**Comment by hvy** [comment]
Hm, do you know why the allennlp test keeps failing after rerunning only the failed jobs. Maybe the latest optuna is not used somehow.

**Comment by himkt** [comment]
`click` is pinned in the extras (https://github.com/optuna/optuna/blob/d529ffe58810c2d4ac1439d98049b175e93122f7/setup.py#L174) but we don't use it to install Optuna. So I had to pin `click` in `requirements.txt` as well.

**Comment by himkt** [comment]
`HEAD is now at 6cf907e Merge 2b286d86b4b3ea9feef9dc3a0677d659584dfbda into 7babe45945ab4c409429012865c85cd81eb2d2d6` Hmm..., revision is still old...

https://github.com/optuna/optuna-examples/runs/5734991333?check_suite_focus=true#step:2:412

And now all checks disappear... 

Screen Shot 2022-03-29 at 19 46 23

Let me close this PR and reopen.

**Comment by himkt** [comment]
Hmm, let me create another PR.

**Pull request closed without merging**